interpreter/prune_test.go

```diff
@@ -194,7 +312,21 @@ func TestPrune(t *testing.T) {
 		state := NewEvalState()
 		reg := newTestRegistry(t)
 		attrs := NewPartialAttributeFactory(containers.DefaultContainer, reg, reg)
-		interp := NewStandardInterpreter(containers.DefaultContainer, reg, reg, attrs)
+		dispatcher := NewDispatcher()
+		dispatcher.Add(functions.StandardOverloads()...)
+		dispatcher.Add(&functions.Overload{
```

Comment:
Similar to `StandardOverloads` above, could we group the optional overloads?

Reply:
Ah, these are not standard overloads (since they're technically defined in a much higher package), but I can make the available for testing via a convenient name.

Reply:
Test method added.